Library/Homebrew/utils/github.rb

```diff
@@ -50,24 +50,34 @@ def api_credentials
       if ENV["HOMEBREW_GITHUB_API_TOKEN"]
         ENV["HOMEBREW_GITHUB_API_TOKEN"]
       elsif ENV["HOMEBREW_GITHUB_API_USERNAME"] && ENV["HOMEBREW_GITHUB_API_PASSWORD"]
-        [ENV["HOMEBREW_GITHUB_API_USERNAME"], ENV["HOMEBREW_GITHUB_API_PASSWORD"]]
+        [ENV["HOMEBREW_GITHUB_API_PASSWORD"], ENV["HOMEBREW_GITHUB_API_USERNAME"]]
```

Comment:
I did in https://github.com/Homebrew/brew/pull/581#issuecomment-234932442 and updated the PR body to reference this comment. What else could/should I have done? (Only noticed the bug after addressing the first feedback.)